pyi.Dockerfile

```diff
@@ -154,17 +153,33 @@ RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
         yum clean all; \
     fi
 
+# python 3.10 installation
+WORKDIR /python
+RUN yum install -y \
+    openssl-devel \
+    bzip2-devel \
+    libffi-devel \
+    perl-core \
+    zlib-devel \
+    xz-devel \
+    wget && \
+    yum groupinstall -y "Development Tools" && \
+    yum clean all
+COPY ./scripts/update_openssl.sh .
```

Comment:
Naming wise - let's follow the naming scheme of other build scripts, so these 2 need to be:
* `openssl_build.sh`
* `python310_build.sh`